extensions/adobe/experience/decisioning/fragmentItem.schema.json

```diff
@@ -42,25 +32,28 @@
           "type": "string",
           "description": "Locale associated with the fragment (eg en_US, en, fr_FR, etc) "
         },
+        "https://ns.adobe.com/experience/decisioning/fragmentitem/fragment-metadata": {
+          "$ref": "https://ns.adobe.com/experience/decisioning/fragment-metadata",
+          "title": "metadata of fragment",
+          "description": "Metadata associated with the AJO Fragment which is stored/published at AJO (outside of ExD)"
+        },
         "https://ns.adobe.com/experience/decisioning/fragmentitem/data": {
-          "title": "data of fragment",
+          "title": "data of fragment.",
           "type": "object",
           "description": "Data associated with the fragment",
-          "additionalProperties": true
-        },
-        "https://ns.adobe.com/experience/decisioning/fragmentitem/publishDate": {
-          "title": "publish date",
-          "type": "string",
-          "format": "date-time",
-          "description": "Publish date & time of the fragment at AJO (outside of ExD)"
-        },
-        "https://ns.adobe.com/experience/decisioning/fragmentitem/publishStatus": {
-          "title": "publish status",
-          "type": "string",
-          "description": "Publish status of the fragment at AJO (outside of ExD)",
-          "meta:enum": {
-            "published": "Published",
-            "un-published": "Un-published"
+          "properties": {
+            "rawData": {
+              "title": "Raw data from fragment",
+              "type": "object",
+              "description": "Raw data from Fragment, which would be useful for offline ranking, etc",
+              "additionalProperties": true
```

Comment:
You have an object with undefined structure in XDM. You will need to define all properties of object.